sources/packages/web/Dockerfile.dev

```diff
@@ -1,5 +1,5 @@
 # "builder" stage, based on Node.js, to build and compile the frontend.
-FROM node:16.17.0-alpine3.16 AS builder
+FROM node:18.17.1-alpine3.18 AS builder
```

Comment:
Question to devs. Shouldn't we change the local docker for web to nginx?

@andrewsignori-aot @ann-aot @guru-aot @cditcher ?

Reply:
nginx:122 doesn't have npm:
![image](https://github.com/bcgov/SIMS/assets/78114138/df1ea68f-b557-4cf3-a090-dcf478e5fb38)

Reply:
Im good. 

https://github.com/bcgov/SIMS/pull/2448#discussion_r1369187397